sdk/communication/Azure.Communication.CallingServer/tests/CallRecording/CallRecordingTests.cs

```diff
@@ -12,7 +12,7 @@ namespace Azure.Communication.CallingServer
     public class CallRecordingTests : CallAutomationTestBase
     {
         private const string RecordingId = "sampleRecordingId";
-        private const string ServerCallId = "sampleServerCallId";
+        private new const string ServerCallId = "sampleServerCallId";
```

Comment:
question: what does this `new` keyword do here?

Reply:
i have a ServerCallId defined in the CallAutomationTestBase class used for CallConnectionTests and CallAutomationClientTests classes. this is to override the field in the CallRecordingTests. Should i remove this field in CallRecordingTests and just use the one in the base class instead?

Reply:
I think that it should work with your own values. Just get rid of it :)